PendingReleaseNotes.md

```diff
@@ -18,6 +18,8 @@
   - Object Stores are defined by a CRD and handled by the Operator
   - Multiple object stores supported through Ceph realms
 - OSDs
+  - Bluestore is now the default backend store for OSDs in a new Rook cluster.  Existing Rook clusters will not be updated or migrated.
```

Comment:
Does this sentence belong? I don't see how it applies to osds and this change
`Existing Rook clusters will not be updated or migrated`

Reply:
I was trying to convey that even though the default backend has changed to Bluestore, we won't touch your existing filestore cluster.  It will continue using filestore.  Perhaps this is already captured by saying "new Rook cluster".